OneCommunityGlobal/HGNRest
# Carlos redo anniversary celebrated

This part of a new Report Page: Create Company Summary Dashboard for Admins requested by Jae.
Implements # (WBS)
Anniversary Celebrared Component

Related PRS (if any):
This Backend PR is related to [FPR#2772 ](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2772)

this PR fix previus [PR#1128](https://github.com/OneCommunityGlobal/HGNRest/pull/1128)
…

Main changes explained:
Visualization of volunteer anniversaries within the current week.
Comparison of volunteer anniversaries between the current and previous week.
Clicking on the volunteer email icon allows you to send a personalized congratulatory email using previously configured -
Gmail OAuth settings. (For more information on Gmail setup, please watch the video below.)
…
How to test:
Check out the current branch.
Do npm install and ... to run this PR locally.
Clear site data/cache.
Log in as admin user.
Navigate to Dashboard > Reports > Total Org Summary > Volunteer Engagement Trends.
Set up the Gmail Api in your account http://console.cloud.google.com/ and https://developers.google.com/ (please watch the Gmail Api setup video below until minut 7).
Set up the environment variables in the .env file of your backend project (see .env image).
Chech the anniversary list.
Select and click on the email icon.
Send a congratulatory email.
Check sent folder of your gmail acount
Screenshots or videos needed:
App features video:

https://www.loom.com/share/d4f75956df7b4ef6946dd17aa8bfa806?sid=fc6227af-88b0-41ec-b0bc-4613367dc07f

Gmail Api setup video (just watch until minut 7)
https://www.youtube.com/watch?v=-rcRf7yswfM&list=LL&index=8&t=304s
